tests/unit/compareBuckets/Injector.js

```diff
@@ -0,0 +1,165 @@
+const async = require('async');
+const assert = require('assert');
+const Prando = require('prando');
+const werelogs = require('werelogs');
+
+werelogs.configure({
+    level: 'info',
+    dump: 'error',
+});
+
+class Injector {
+
+    constructor(backend) {
+        this.backend = backend;
+        this.nOps = 0;
+        this.nPuts = 0;
+        this.nDeletes = 0;
+        this.rnd = new Prando(0);
+        this.logger = new werelogs.Logger('Injector');
+    }
+
+    printStats() {
+        // eslint-disable-next-line
+        console.error('nOps', this.nOps);
+        // eslint-disable-next-line
+        console.error('nPuts', this.nPuts);
+        // eslint-disable-next-line
+        console.error('nDeletes', this.nDeletes);
+    }
+
+    genKey(len) {
+        let result = '';
+        const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+        const charactersLen = characters.length;
+        for (let i = 0; i < len; i++) {
+            result += characters.charAt(
+                Math.floor(
+                    this.rnd.next() *
+                        charactersLen));
+        }
+        return result;
```

Comment:
prando seems to have a next string method, maybe something like:
```suggestion
        const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
        return this.rnd.nextString(len, characters);
```

Reply:
Yes, did not see it. Prando is really cool because it can be seeded, so make hard bugs to find easily reproducible.